litr/src/main/java/com/linkedin/android/litr/render/FrameDropper.kt

```diff
@@ -0,0 +1,36 @@
+package com.linkedin.android.litr.render
+
+/**
+ * A simple video frame dropper which provides a [shouldRender] function. May be used when the
+ * target video should have a lower frame rate than its source video.
+ *
+ * @see <a href="https://github.com/linkedin/LiTr/issues/120#issuecomment-859318538">Suggested by @alexvasilkov</a>
+ */
+internal interface FrameDropper {
+
+    fun shouldRender(): Boolean
+
+    companion object {
+
+        fun getDefaultFrameDropper(
```

Comment:
Why do we need this static factory method instead of just having a `DefaultFrameDropper` implementation and creating it directly in `TrackTranscoder`?

Reply:
Good question, don't have a valid answer for that 😅. Updated it at: [FrameDropper.kt#L14-L33](https://github.com/linkedin/LiTr/blob/a6d524823017dc2cf30ed85026e7d1ad3af4ace9/litr/src/main/java/com/linkedin/android/litr/render/FrameDropper.kt#L14-L33)

Reply:
Good question, don't have a valid answer for that 😅. Updated it at: [FrameDropper.kt#L14-L33](https://github.com/linkedin/LiTr/blob/a6d524823017dc2cf30ed85026e7d1ad3af4ace9/litr/src/main/java/com/linkedin/android/litr/render/FrameDropper.kt#L14-L33)